src/Frontend.hs

```diff
@@ -262,7 +258,6 @@ aulaTesting =
   :<|> batchCreateUsers
   :<|> (PageShow <$> Action.persistent mkRandomPassword)
 
-
```

Comment:
(sorry about the long-lines-with-minusses.  they should never be there.  i started that because there haddocks section headings need to be in the explicit export list if there is one, not in the code itself.  so @Mikolaj you were technicall / legally doing the right thing deleting that empty line.)